internal/domain/review_suggestion.go

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import (
+	"github.com/invopop/jsonschema"
+)
+
+type ReviewSuggestion struct {
+	Comment  string `json:"comment"`
+	Position int    `json:"position"`
+}
```

Comment:
The `GenerateSchema` function creates a new reflector each time it's called, which may not be efficient. To improve performance, consider declaring the `reflector` variable as a global variable if its configuration remains constant, thus avoiding repetitive creation. Alternatively, keep the function as is unless performance profiling identifies it as a bottleneck.